pkg/errors/sentry.go

```diff
@@ -22,7 +22,7 @@ func CaptureError(err error, tags map[string]string) {
 }
 
 func CaptureDiagnostics(dd diag.Diagnostics, tags map[string]string) {
-	allowUnmanaged := core.Version == core.DevelopmentVersion && viper.GetBool("debug-sentry")
+	allowUnmanaged := viper.GetBool("debug-sentry")
```

Comment:
`core` reference removed because we shouldn't care which binary we're on if `--debug-sentry` is specified? (or we can embed `core.Version == core.DevelopmentVersion` as a bool inside the fetch diag as a fourth arg... which also kinda works)